app/views/application/_navbar.html.haml

```diff
@@ -0,0 +1,24 @@
+%nav.navbar.navbar-inverse
+  .container-fluid
+    .navbar-header
+      %button.navbar-toggle.collapsed{"aria-controls" => "navbar", "aria-expanded" => "false", "data-target" => "#navbar", "data-toggle" => "collapse", :type => "button"}
+        %span.sr-only Toggle navigation
+        %span.icon-bar
+        %span.icon-bar
+        %span.icon-bar
+      = link_to 'Best Slow Food', root_path, class: 'navbar-brand'
+    #navbar.navbar-collapse.collapse
+      %ul.nav.navbar-nav
+        %li
+          = link_to "Home", root_path
+        %li
+          = link_to "Create dish", new_dish_path
+        %li
+          = link_to 'Show cart', carts_path
+        %li
+          = link_to 'First dish in system', dish_path(1)
+      %ul.nav.navbar-nav.navbar-right
+        %li
+          = link_to 'Log in', new_user_session_path
+        %li
+          = link_to 'Sign up', new_user_registration_path
```

Comment:
I saw that you added that task to that story (Fix sign_out routing to users can sign out) That chore is about something else. Conditioning these links are either a separate feature or should be fixed in this one (I would do it in this one personally)